src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.FileSystem.cs

```diff
@@ -91,6 +92,7 @@ private static string[] GetValueNamesCore(int values)
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_Registry);
         }
 
+        [return: NotNullIfNotNull("defaultValue")]
```

Comment:
Does the repo use a compiler that supports `nameof` of the parameter name?

Reply:
Huh, I didn't think we were, but apparently we are.  Fixed.  And I'll submit a PR to fix up existing usage.